src/data/entity/user.entity.ts

```diff
@@ -1,16 +1,26 @@
+import { Field, ID, ObjectType } from 'type-graphql';
 import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
 
 @Entity()
+@ObjectType()
```

Comment:
Opa, pensando em arquitetura, aqui o ideal seria ter outro objeto para a resposta (em outro arquivo e pasta, inclusive). 
Não é uma boa reaproveitar a mesma classe do ORM para a resposta do GraphQL porque fica muito acoplado (e se você for salvar a data do banco mas precisar retornar a data? já começa a ficar meio embaralhado o negócio ou até mesmo campos que é para ter no banco mas não é para retornar no graphql, como a senha)

Ou ainda considerando que o objeto de resposta poderia ser composto por mais informações que o UserEntity (supondo que precise consultar um serviço externo para pegar mais informações do cadastro ou computar alguma coisa para preencher outro campo)